build.gradle.kts

```diff
@@ -12,8 +13,13 @@ description = "A Gradle plugin for generating build info as Java code."
 
 java { toolchain.languageVersion.set(JavaLanguageVersion.of(11)) }
 
+buildInfo {
+    packageName.set("com.opencastsoftware.gradle.buildinfo")
+    properties.set(mapOf("javaPoetVersion" to libs.versions.javaPoet.get()))
+}
+
 dependencies {
-    implementation(libs.javaPoet)
+    compileOnly(libs.javaPoet)
```

Comment:
Using a `compileOnly` dependency enables us to compile our code without pulling it into users' build classpaths.